### PR TITLE
feat: 1.10 release notes

### LINKED ISSUE #6886: Release Notes for 1.10.0
### Is your feature request related to a problem? Please describe

Create Release Notes for 1.10.0

### Describe the solution you'd like

do release notes

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by deboer-tim**
### What does this PR do?

Draft release notes for 1.10.

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

Fixes #6886.

### How to test this PR?

`yarn website:dev`

### CODE REVIEW

File: website/blog/2024-04-30-release-1.10.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+who has helped get us to this number - especially those who have opened issues or PRs and contributed
+to this success!
+
+Looking forward to the next million! 🚀
+
+### Extension Catalog
+
+To make it easier to access extensions we added an item to the main navigation and completely revamped
+the Extensions page. You can now see and control all your Podman Desktop and Docker Desktop extensions
+in a single place, and if you want more details just click on an extension to see the readme.
```
Comment (cdrage) [inline website/blog/2024-04-30-release-1.10.md]:
```suggestion
in a single place. If you want to view more details, click on an extension to see the expanded readme.
```

File: website/blog/2024-04-30-release-1.10.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+![extensions](img/podman-desktop-release-1.10/extensions.png)
+
+With the growing number of extensions available we've also introduced a catalog to make it easy to find
+and install them. Check this page regularly as we hear there are some exciting new extensions sailing in!
```
Comment (cdrage) [inline website/blog/2024-04-30-release-1.10.md]: Check this page = check which page?

File: website/blog/2024-04-30-release-1.10.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+![extension catalog](img/podman-desktop-release-1.10/extension-catalog.png)
+
+### Podman v5 (5.0.2)
+
+Now that we've had some experience and time to test our migration tools, we're glad to make Podman v5 a
+recommended upgrade for all users. Podman Desktop will help ensure you don't lose any data during the
+migration, and macOS users in particular will enjoy the performance enhancements of
```
Comment (cdrage) [inline website/blog/2024-04-30-release-1.10.md]:
```suggestion
migration. In particular, macOS users will see a performance boost as we've switched 
```

File: website/blog/2024-04-30-release-1.10.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+moving to Apple Hypervisor.
```
Comment (cdrage) [inline website/blog/2024-04-30-release-1.10.md]:
```suggestion
to the native Apple Hypervisor.
```

File: website/blog/2024-04-30-release-1.10.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+In case you missed it, the blog announcement for 🦭 Podman version 5 is [here](https://blog.podman.io/2024/03/podman-5-0-has-been-released/).
```
Comment (cdrage) [inline website/blog/2024-04-30-release-1.10.md]:
```suggestion
Find out more details on the blog announcement for 🦭 Podman version 5 [here](https://blog.podman.io/2024/03/podman-5-0-has-been-released/).
```

File: website/blog/2024-04-30-release-1.10.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+### Extension API Improvements
+
+We added support for manifests (multi-arch images) to the extension API to give upcoming extensions
```
Comment (cdrage) [inline website/blog/2024-04-30-release-1.10.md]: We've? I think?

File: website/blog/2024-04-30-release-1.10.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+🎉 We’d like to say a big thank you to everyone who helped make 🦭 Podman Desktop even better. In this release we received pull requests from the following people:
+
+- [Norman Dankert](https://github.com/totkeks) in [feat: open dashboard with left click on tray icon on Windows](https://github.com/containers/podman-desktop/pull/6750)
+- [Tim Heuer](https://github.com/timheuer) in [docs: changes to writing an extension](https://github.com/containers/podman-desktop/pull/6715)
+- [Hlib Haranin](https://github.com/GLEF1X) in [feat(webview-telemetry): make telemetry for webviews more detailed](https://github.com/containers/podman-desktop/pull/6692) and [feat(container-detail): make image name a link that redirects to image detail page](https://github.com/containers/podman-desktop/pull/6628)
+- [tmancill](https://github.com/tmancill) in [docs: Update pushing-an-image-to-kind.md - add info re: crictl](https://github.com/containers/podman-desktop/pull/6666)
+
+---
+
+## Final notes
```
Comment (cdrage) [inline website/blog/2024-04-30-release-1.10.md]:
```suggestion
## Final Notes
```

**Review by cdrage — CHANGES_REQUESTED** [review]
Great release notes! Only a few changes requested.

File: website/blog/2024-04-30-release-1.10.md (hunk 1 above)
Comment (deboer-tim) [reply website/blog/2024-04-30-release-1.10.md]: Fixed.

File: website/blog/2024-04-30-release-1.10.md (hunk 3 above)
Comment (deboer-tim) [reply website/blog/2024-04-30-release-1.10.md]: Fixed.

File: website/blog/2024-04-30-release-1.10.md (hunk 4 above)
Comment (deboer-tim) [reply website/blog/2024-04-30-release-1.10.md]: Fixed.

File: website/blog/2024-04-30-release-1.10.md (hunk 6 above)
Comment (deboer-tim) [reply website/blog/2024-04-30-release-1.10.md]: Fixed.

File: website/blog/2024-04-30-release-1.10.md (hunk 7 above)
Comment (deboer-tim) [reply website/blog/2024-04-30-release-1.10.md]: Fixed.

File: website/blog/2024-04-30-release-1.10.md (hunk 5 above)
Comment (deboer-tim) [reply website/blog/2024-04-30-release-1.10.md]: Fixed.

File: website/blog/2024-04-30-release-1.10.md (hunk 2 above)
Comment (deboer-tim) [reply website/blog/2024-04-30-release-1.10.md]: Fixed (catalog).

**Review by cdrage — APPROVED** [review]

File: website/blog/2024-04-30-release-1.10.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+
+Podman Desktop 1.10 Release! 🎉
+
+![Podman-desktop-1-10-hero](img/podman-desktop-release-1.10/1000000-downloads.png)
+
+This release introduces:
+
+- **1 Million Downloads!**: Wow, we made it!
+- **Extension Catalog**: Redesigned extensions page and catalog to get the most out of Podman Desktop.
+- **Podman 5** [Podman 5.0.2](https://github.com/containers/podman/releases/tag/v5.0.2) now recommended for all users.
```
Comment (axel7083) [inline website/blog/2024-04-30-release-1.10.md]:
```suggestion
- **Podman 5**: [Podman 5.0.2](https://github.com/containers/podman/releases/tag/v5.0.2) now recommended for all users.
```

**Review by axel7083 — APPROVED** [review]

File: website/blog/2024-04-30-release-1.10.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+- feat: add inspectManifest API endpoint [#6812](https://github.com/containers/podman-desktop/pull/6812)
+- feat: add createManifest API [#6630](https://github.com/containers/podman-desktop/pull/6630)
+
+---
+
+## Other Notable Enhancements
+
+We've added lots of features this release, here are some other highlights:
+
+- feat: add multi-arch building / manifest creation [#6792](https://github.com/containers/podman-desktop/pull/6792)
```
Comment (slemeur) [inline website/blog/2024-04-30-release-1.10.md]: don't we think this is worth giving a dedicated paragraph?

Comment (deboer-tim) [reply website/blog/2024-04-30-release-1.10.md]: Yes, we should. Multi-platform section and image added in new commit.

**Review by cdrage — APPROVED** [review]

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
as we released 1.10.1 and 1.10.2 I think the number of fixed issues is bigger than what is listed in the release notes

**Comment by deboer-tim** [comment]
> as we released 1.10.1 and 1.10.2 I think the number of fixed issues is bigger than what is listed in the release notes

Added most of them / the ones I thought were notable.

**Pull request merged**